google/cloud/google_cloud_cpp_common.cmake

```diff
@@ -214,6 +214,34 @@ install(
     DESTINATION "${CMAKE_INSTALL_LIBDIR}/cmake/google_cloud_cpp_common"
     COMPONENT google_cloud_cpp_development)
 
+# Create a header-only library for the mocks. We use a CMake `INTERFACE` library
+# for these, a regular library would not work on macOS (where the library needs
+# at least one .o file). Unfortunately INTERFACE libraries are a bit weird in
+# that they need absolute paths for their sources.
+set(relative_mock_files mocks/mock_stream_range.h)
```

Comment:
nit: should we add the magic comment to keep the "list" of mocks sorted?

Reply:
Done.